cmd/kubeadm/app/util/system/kernel_validator.go

```diff
@@ -78,7 +78,7 @@ func (k *KernelValidator) Validate(spec SysSpec) (error, error) {
 
 // validateKernelVersion validates the kernel version.
 func (k *KernelValidator) validateKernelVersion(kSpec KernelSpec) error {
-	glog.Infof("Validating kernel version")
+	glog.V(1).Infof("Validating kernel version")
```

Comment:
Why to use Infof here? We're not formatting anything, so just glog.V(1).Info should be ok.